config/initializers/constants.rb

```diff
@@ -19,5 +19,15 @@
   'Other'
 ].freeze
 
+ALL_PROJECT_STATUS = [
+  'Idea',
+  'Brainstorming',
+  'Prototyping',
+  'In development',
+  'In production',
+  'Live',
+  'In the wild'
```

Comment:
Since some of these statuses kind of overlap, let's simplify this list to just:
`Idea`
`Prototyping`
`In development`
`Live`

Hard to remove statuses after the fact, but easier to add new ones later.

Reply:
Done